bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/FormUtil.java

```diff
@@ -0,0 +1,23 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2024 Adobe
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+package com.adobe.cq.forms.core.components.models.form;
+
+import org.osgi.annotation.versioning.ProviderType;
+
+@ProviderType
+public interface FormUtil {
```

Comment:
Let's rename this to HTLUtils. Form is already there in the package name, hence form in the class name is redundant

Reply:
I feel FormUtils will allow us to write more wider utility methods whereas renaming it to HTLUtils will require creating new utility models if its not required in HTL. Although i can't think of a reason of creating an Util as a Sling model if its not required in HTL.

So renaming this file to HtlUtil